content/fr/es6.md

```diff
@@ -1,23 +1,22 @@
-# ES6 on io.js
+# ES6 et io.js
 
-io.js is built against modern versions of [V8](https://code.google.com/p/v8/). By keeping up-to-date with the latest releases of this engine we ensure new features from the [JavaScript ECMA-262 specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm) are brought to io.js developers in a timely manner, as well as continued performance and stability improvements.
+io.js est construit sur des versions modernes de [V8](https://code.google.com/p/v8/). En restant à jour avec les dernières livraisons du moteur, nous nous assurons que les dernières fonctionnalités de la [spécification JavaScript ECMA-262](http://www.ecma-international.org/publications/standards/Ecma-262.htm) sont mises à disposition des développeurs io.js en continu, ainsi que des améliorations de performance et de stabilité.
 
-Version 1.1.0 of io.js ships with V8 4.1.0.14, which includes ES6 features well beyond version 3.26.33 that will be shipped with joyent/node@0.12.x.
+La version 1.1.0 d'io.js est livrée avec V8 4.1.0.14 qui inclue des fonctionnalités ES6 qui vont bien au delà de la version 3.26.33 qui sera fournie avec joyent/node@0.12.x.
```

Comment:
This line has changed : https://github.com/iojs/website/blob/master/content/en/es6.md

We'll need to figure out a way to keep our translations in sync with the original.